docs/faq.md

```diff
@@ -580,10 +622,11 @@ You can do that via the modifiers.ini file. Below is not a comprehensive list of
 No, Multitroid is a mod made by MilesTheNerd. While we greatly enjoy the mod and Miles is part of our team, it is not officially supported at present due to the time and energy required to provide official multiplayer support.
 
 #### How do I host on Multitroid?  
-1. You need to have the AM2RServer downloaded: <https://github.com/lassiterm/AM2R-Server/releases/latest>
-2. You need to either port forward, or use a VPN (like Hamachi, ZeroTier, Radmin etc.).
+1. Install [AM2RServer](https://github.com/lassiterm/AM2R-Server/releases/latest)
```

Comment:
A) having the link visible, makes it less likely that people are gonna miss to download it
B) use "download" instead of "install" please.

Reply:
My bad, I'll keep that in mind next time. I'll undo the link.

Reply:
Done